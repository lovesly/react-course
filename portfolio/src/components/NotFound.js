import React from 'react';
import { Link } from 'react-router-dom';

const NotFoundPage = () => (
    <div>
        Getting Lost...
        <Link to="/">Go home</Link>
    </div>
);

export default NotFoundPage;