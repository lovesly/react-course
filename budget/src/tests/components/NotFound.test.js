import React from 'react';
import { shallow } from 'enzyme';
import NotFound from '../../components/NotFound';


test(`should render expenselist with expenses`, () => {
    const wrapper = shallow(<NotFound />);
    expect(wrapper).toMatchSnapshot();
});