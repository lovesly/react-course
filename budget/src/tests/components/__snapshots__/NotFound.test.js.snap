// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render not found page correctly 1`] = `
<div>
  Getting Lost...
  <Link
    replace={false}
    to="/"
  >
    Go home
  </Link>
</div>
`;
